### PR TITLE
10.0.1 release

### PR DESCRIPTION
**Pull request by mjhenkes**
Open PR tagged for this release: 

[#21994 Inline Specs List in 10.0 Runner is not scrollable ](https://github.com/cypress-io/cypress/pull/21994)

### CODE REVIEW

**Review by flotwig — APPROVED** [review]

**Review by jordanpowell88 — APPROVED** [review]

**Review by jordanpowell88 — DISMISSED** [review]
We need to wait for https://github.com/cypress-io/cypress/issues/22002 and add that here per @jennifer-shehane

**Comment by jennifer-shehane** [comment]
@jordanpowell88 Yes, there's a few more PRs we will wait for here.

File: content/_changelogs/10.0.1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+_Released 6/1/2022_
+
+**Bugfixes:**
+
+- Corrected an issue where the inline spec list is not scrollable. Fixed
+  [#21995](https://github.com/cypress-io/cypress/issues/21995)
+- Corrected an issue where the `.check()` and `.uncheck()` would not remove the
+  indeterminate state from checkboxes. Fixed
+  [#21665](https://github.com/cypress-io/cypress/pull/21665)
+- Corrected aa issue where a direct download of Cypress would throw an error on
```
Comment (flotwig) [inline content/_changelogs/10.0.1.md:12]:
```suggestion
- Corrected an issue where a direct download of Cypress would throw an error on
```

File: content/_changelogs/10.0.1.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  loading in a fresh project. Fixed
```
Comment (flotwig) [inline content/_changelogs/10.0.1.md:13]:
```suggestion
  migrating a project without `cypress` installed. Fixed
```

**Pull request merged**